zerver/lib/streams.py

```diff
@@ -146,10 +146,19 @@ def create_streams_if_needed(
     added_streams: List[Stream] = []
     existing_streams: List[Stream] = []
     for stream_dict in stream_dicts:
+        is_invite_only = stream_dict.get("invite_only", False)
```

Comment:
Is there a reason not to call this variable just `invite_only`?  That seems more consistent.